tuts/create-your-first-substrate-chain/v2.0.0-alpha.7/setup.md

```diff
@@ -27,14 +27,11 @@ of those prerequisites installed for you:
 curl https://getsubstrate.io -sSf | bash -s -- --fast
 ```
 
+You can view the source code for this script by visiting [https://getsubstrate.io](https://getsubstrate.io) in your browser.
+
 > If you did not have Rust installed prior to running this script, make sure to add restart your
 > terminal before continuing (command given in last line of the script output).
 
-<details>
```

Comment:
Another good catch! I was just removing the tags altogether and guess I just missed this one somehow :sweat_smile: I figured it was the "right thing" to do because we don't want HTML tags but wasn't sure if it was how we wanted the actual page to be formatted. Still not sure on that one.

Reply:
Removed the closing tag but leaving this conversation unresolved because I'm not sure if this is actually how we want to display this information.

Reply:
If we are sticking to the hard rule that no html should be in the markdown files, that I guess this is the best option.